cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/lib.rs

```diff
@@ -0,0 +1,1077 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # Bridge Hub Westend Runtime
+//!
+//! This runtime currently supports bridging between:
+//! - Rococo <> Westend
```

Comment:
I don't understand:
- this PR adds `Westend` flavor to `cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo` runtime so that BHs can be deployed from same (rococo-based) runtime on `Rococo`, `Wococo` **and** `Westend`,
- then this PR also adds dedicated `cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend` runtime files

### suggestions

1. Unless explicitly needed for `Westend` parachains emulation tests or somewhere else, I would drop this dedicated `bridge-hub-westend-runtime` package/crate (helps with compile of `polkadot-parachains-bin` too).
2. If however, we do need dedicated `bridge-hub-westend-runtime` crate, then we should drop the flavors stuff in `bridge-hub-rococo-runtime` and eliminate complexity there.

if we do `1.` then please do it in this PR (should also drastically reduce its size); if we do `2.`, you can do it in follow-up PR that also completely removes `Wococo` (flavor and references)

Reply:
maybe @joepetrowski has a more informed preference between `1.` and `2.` here

Reply:
> this PR adds Westend flavor to cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo runtime

Actaully, no, it does not. I wanted to do that with flavor (it was almost done that way), but then I switched it to the clean `bridge-hub-westend` which is much cleaner. Also they are adding [`collectives-westend`](https://github.com/paritytech/polkadot-sdk/pull/2024). So There wont be any confusion.
asset-hub-rococo
bridge-hub-rococo
(collectives-rococo (maybe one day))
asset-hub-westend
bridge-hub-westend
collectives-westend

Yes, when Rococo<>Westend is deployed and working, I will remove all flavor Wococo stuff and drop all VMs and whatever related to the Wococo.

Reply:
We are always discussing this tradeoff of more runtimes vs. duplicated code. We tried to use production runtimes in testnets (like Collectives Polkadot on Westend), but then you can't really test things there first. Maintaining "first class" runtimes for Rococo and Westend seems not unreasonable, with others (Wococo, Versi, etc.) being special cases.